core/CMake_readelf.cmake

```diff
@@ -40,9 +40,12 @@
 # PR 212290: ensure no text relocations (they violate selinux execmod policies)
 # looking for dynamic section tag:
 #   0x00000016 (TEXTREL)                    0x0
+
+include(../${lib_fileloc}.cmake)
```

Comment:
Please list among the inputs "Caller must set" above.

Make it clear this is both the file name *and* the variable name that the file sets.  That seems weird: I would expect the file name to be an absolute path.  Explain precisely what kind of path and what assumptions on where it's located.  Ditto for all the other .cmake files taking in "lib_fileloc".

Reply:
Done.